script/Ciel/Day 11/11_01_CIEL11_1D.txt

```diff
@@ -221,5 +225,5 @@ And no more pesky <Executor|w o m a n>.
 [sha:ac3bb1e89cc4fd2fad8148b9b093e43e23c81493]{
 -- Page 98, Offset 29327.
 -- 俺は深く息を吐いて、深い眠りに落ちる事にした。
-Letting out a deep breath, I prepare to fall into a deep slumber.
+I let out a deep breath, and sink into a bottomless slumber.
```

Comment:
bottomless? I don't think that's a very common collocation with slumber. Just go with deep.

Reply:
It's less common, but it does make sense and I hate double-using adjectives (there's already a deep in this line)

Reply:
Heavy, then.

Reply:
I'm honestly baffled that this apparently isn't a common collation. I still think it works better than heavy though, since that doesn't seem too common either and bottomless pairs well with sinks.

Reply:
I mean, bottomless is one of those words used to describe things that usually have bottoms? It isn't that strange.
To me it just feels kinda out of place so I'd rather use something that doesn't feel vaguely wrong to me.

Long breath - deep slumber is another variation, because then you retain the feeling of depth to the 'plunge' (which is what the original is going for anyway).